spec/unit/filter_spec.rb

```diff
@@ -317,6 +317,29 @@ def expect_file_scope_filter_change(scope, filter, before, after)
 EOF
       expect(captured_machinery_output).to include(expected_output)
     end
+
+    it "raises error when given wrong path" do
+      expect {
+        expect_file_scope_filter_change(
+          "changed_managed_files",
+          Machinery::Filter.new(["/changed_managed_files/files/name=element_a,element_b"]),
+          [
+            "/etc/deleted changed managed",
+            "/etc/cron.d",
```

Comment:
I would use less examples, two are more than enough because they are not causing the issue. Even one line would be Ok.